src/obfl-specification.html

```diff
@@ -1628,7 +1684,8 @@ <h4 id="L9241" class="include">toc-sequence</h4>
         <dt>toc [required]</dt>
           <dd>A reference to a table-of-contents</dd>
         <dt>range [required]</dt>
-          <dd>A data selection property (document/volume)</dd>
+          <dd>A data selection property (document/volume/document-resumed/volume-resumed)</dd>
```

Comment:
Not sure if we discussed this change or if it's strictly needed? A document that has resumed elements should be handled. If you don't want them then don't add them. Right?

Reply:
In the discussion that @bertfrees and I had after your visit in Grave, Bert requested the option to just disregard the resumed functionality. He thought it might be convenient. That's why I have added these options here.

Reply:
Yes, or at least Paul and I did. I can't find the logs of that conversation anymore and I don't remember what the outcome was, but my position was that while new values for range _could_ be added, it isn't _needed_. Indeed, the simple solution is: if you don't want them then don't add them. Paul's argument for adding the new ranges is that you could avoid some duplication this way. If we add the new values, we should definitely call them differently. These names aren't very self-explaining. But I wouldn't add them at all.

Reply:
So the conclusion is to remove these?

Reply:
I'll remove them.